<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(\&quot;loginwallpaper.jpg\&quot;); background-position: center; </string>
  </property>
  <widget class="QPushButton" name="button_login">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_username">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_create_account">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_hostname">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_connect_direct">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <zorder>line_username</zorder>
  <zorder>line_password</zorder>
  <zorder>button_login</zorder>
  <zorder>button_create_account</zorder>
  <zorder>label_status</zorder>
  <zorder>line_hostname</zorder>
  <zorder>button_connect_direct</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
